api/client/commands.go

```diff
@@ -67,7 +67,7 @@ func (cli *DockerCli) CmdHelp(args ...string) error {
 	if len(args) > 0 {
 		method, exists := cli.getMethod(args[0])
 		if !exists {
-			fmt.Fprintf(cli.err, "docker: '%s' is not a docker command. See 'docker --help'.\n", args[0])
+			fmt.Fprintf(cli.err, "Docker: '%s' is not a docker command. See 'docker --help'\n", args[0])
```

Comment:
Specifically for this one, I think we should keep the lowercasing by convention (as this is really the binary name rather than the project name).